src/shims/unix/linux/epoll.rs

```diff
@@ -39,11 +71,42 @@ impl FileDescription for Epoll {
     fn close<'tcx>(
         self: Box<Self>,
         _communicate_allowed: bool,
+        _ecx: &mut MiriInterpCx<'tcx>,
     ) -> InterpResult<'tcx, io::Result<()>> {
         Ok(Ok(()))
     }
 }
 
+/// The table of all epoll_events.
+pub struct EpollEventTable(BTreeMap<usize, Vec<Weak<RefCell<EpollEvent>>>>);
```

Comment:
Need to remove table entries when file descriptors are removed from the fdtable

Need to remove elements from the `Vec` when they become dangling.

Reply:
probably should check this table in the memory leak detector, too.

Reply:
I opened an issue about this, as that is preexisting for regular file descriptors: https://github.com/rust-lang/miri/issues/3771